ui/app/components/app-footer/index.hbs

```diff
@@ -0,0 +1,5 @@
+<footer id="footer">
+  {{inline-svg "hashicorp" class="icon"}}
+  <span class="copyright">&copy;{{current-year null}} HashiCorp</span>
```

Comment:
```suggestion
  <span class="copyright">&copy; {{current-year null}} HashiCorp</span>
```
This is minor but there's usually whitespace here I think (https://demo.consul.io/ as reference).